static/app/components/group/groupSummary.tsx

```diff
@@ -167,13 +179,17 @@ export function GroupSummary({
       icon: <IconSpan size="sm" />,
       showWhenLoading: false,
     },
-    {
-      id: 'possible_cause',
-      title: t('Possible cause'),
-      insight: data?.possibleCause,
-      icon: <IconFocus size="sm" />,
-      showWhenLoading: true,
-    },
+    ...(shouldShowPossibleCause
+      ? [
+          {
+            id: 'possible_cause',
+            title: t('Possible cause'),
+            insight: data?.possibleCause,
+            icon: <IconFocus size="sm" />,
+            showWhenLoading: false,
```

Comment:
@roaga double-checking this makes sense?

Reply:
ok nvm, this is fine